src/vaadin-date-time-picker.html

```diff
@@ -522,7 +519,6 @@
             // Synchronize properties from slotted time picker
             this.timePlaceholder = timePicker.placeholder;
             this.step = timePicker.step;
-            this.invalid = timePicker.invalid;
             // Don't set disabled or readonly from time picker since they're already set from date picker
```

Comment:
This commend is now outdated and should be removed.

Reply:
Removed.